src/components/DiagnosticsBody.vue

```diff
@@ -83,7 +84,7 @@ export default Vue.extend({
       this.$store.dispatch('diagnostics/updateDiagnostic', { isMuted, row });
     },
     toggleMute() {
-      this.showMuted = !this.showMuted;
+      this.$store.dispatch('preferences/setShowMuted', !this.showMuted as boolean);
```

Comment:
I'm confused — shouldn't `!x` always be a boolean?
(If TypeScript complains, we can't do much about it I guess…)

Reply:
Yes, we get typescript complaints because `mapGetters` doesn't really preserve type information..